### PR TITLE
Fix spelling error

### PR DESCRIPTION
**Pull request by nachtien**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -39,7 +39,8 @@ You can also try the latest mainline build containing bleeding edge features and
    - virocore.aar (for ViroCore)
    - ios_dist.tgz (for iOS)
 
-For ViroCore Android, HelloWorld samples should have the corresponding file at the location viro_core/virocore-release-*version*. As such, simply download the virocore.aar artifact from the workflow above, and then rename (make sure it matches your code sample's name) and replace the virocore-release-v_x_xx_x.aar file in your HelloWorld project
+For ViroCore Android, 
+orld samples should have the corresponding file at the location viro_core/virocore-release-*version*. As such, simply download the virocore.aar artifact from the workflow above, and then rename (make sure it matches your code sample's name) and replace the virocore-release-v_x_xx_x.aar file in your HelloWorld project
```
Comment (adelarsq) [inline README.md:43]: @nachtien Please take a look on this line. Overall is good! Thanks for the contribution!

**Review by adelarsq — CHANGES_REQUESTED** [review]
It's necessary a little change. Overall is good to go. Nice!

File: README.md (hunk 1 above)
Comment (nachtien) [reply README.md]: ahh, i probably shouldn't use the github web editor to make changes _ever_.

**Comment by adelarsq** [comment]
@nachtien Thanks for the contribution!

**Pull request merged**